subgraphs/abracadabra/config/bnb.json

```diff
@@ -0,0 +1,11 @@
+{
+  "startBlock": "12750809",
+  "network": "bnb",
```

Comment:
does this actually work? Shouldn't it be `bsc`?

Reply:
just updated this